docs/docs/typescript/strict-mode.md

```diff
@@ -0,0 +1,239 @@
+---
+description: TS Strict mode tips and tricks
+---
+
+# TypeScript Strict Mode
+
+Looks like you're ready to level up your TypeScript game!
+Redwood supports [strict mode](https://www.typescriptlang.org/docs/handbook/2/basic-types.html#strictness), but doesn't enable it by default.
+While strict mode gives you a lot more safety, it makes your code a bit more verbose and requires you to make small manual changes if you use the generators.
+
+## Enabling strict mode
+
+Enable strict mode by setting `strict` to true in `web/tsconfig.json` and `api/tsconfig.json`:
```

Comment:
Should we mention `scripts/tsconfig.json` too?

Reply:
Sure